lib/platform/azure/index.md

```diff
@@ -0,0 +1,7 @@
+# Azure DevOps and Azure DevOps Server
+
+## Features awaiting implementation
+
+- The `automergeStrategy` configuration option has not been implemented for this platform, and all values behave as if
```

Comment:
Can you create an issue for this instead? Otherwise I'm never going to be able to find this, and it'll never get implemented.

Reply:
Instead of, or in addition to?

Reply:
You caught my edit 😄 I initially thought instead of, but I think in addition to would be better. Thank you!

Reply:
This has been formally requested in #10851